lib/presentation/pages/ready_for_session_page.dart

```diff
@@ -31,13 +32,20 @@ class ReadyForSessionPage extends StatelessWidget {
         text = "So much energy 🔋⚡🔋⚡🔋⚡🔋\nLet's begin";
     }
 
-    return PageEnclosureMolecule(
+     return PageEnclosureMolecule(
       scaffold: false,
       title: 'start_session',
       child: ReadyForSessionOrganism(
-        callback,
+        () => handleCallback(),
         response: text,
       ),
     );
   }
+
+  void handleCallback() => _handleCallback();
+
+  void _handleCallback() {
+    VibrationController.instance.vibrate(VibrationType.light);
+    callback();
+  }
```

Comment:
There is no reason to call function to call function??.

Also move it above the `build` method